### PR TITLE
COM-12448: Add H264 MVC test suite

### PR DESCRIPTION
**Pull request by mdimopoulos**

### CODE REVIEW

File: scripts/gen_jvt.py [hunk 1]
```diff
@@ -244,22 +272,24 @@ def remove_r1_from_path(path):
...
-        "JVT SVC version 1",
+        "JVT Multiview Video Coding test suite",
         H264_URL,
-        True
+        True,
+    )
+    generator.generate(not args.skip_download, args.jobs)
+
+    generator = JVTGenerator(
+        "SVC", "JVT-SVC_V1", Codec.H264, "JVT SVC version 1", H264_URL, True
```
Comment (rsanchez87) [inline scripts/gen_jvt.py:292]: Only a minor change, I don't understand why one enum is separated by line breaks and others are not...

File: scripts/gen_jvt.py [hunk 2]
```diff
@@ -173,18 +186,27 @@ def generate(self, download, jobs):
                         "PPH444P6_Mitsubishi_A": OutputFormat.NONE,
                         "PPH444P7_Mitsubishi_A": OutputFormat.NONE,
                         "PPH444P8_Mitsubishi_A": OutputFormat.NONE,
-                        "PPH444P9_Mitsubishi_A": OutputFormat.NONE
+                        "PPH444P9_Mitsubishi_A": OutputFormat.NONE,
```
Comment (rsanchez87) [inline scripts/gen_jvt.py:189]: Why the last "," ?

File: scripts/gen_jvt.py [hunk 3]
```diff
@@ -269,6 +299,6 @@ def remove_r1_from_path(path):
         Codec.H264,
         "JVT professional profiles version 1",
         H264_URL,
-        True
+        True,
```
Comment (rsanchez87) [inline scripts/gen_jvt.py:302]: Why the last "," ?

File: scripts/gen_jvt.py [hunk 4]
```diff
@@ -81,7 +81,7 @@ def __init__(
         codec: Codec,
         description: str,
         site: str,
-        use_ffprobe: bool = False
+        use_ffprobe: bool = False,
```
Comment (rsanchez87) [inline scripts/gen_jvt.py:84]: Why the last "," ?

File: scripts/gen_jvt.py [hunk 5]
```diff
@@ -146,11 +151,19 @@ def generate(self, download, jobs):
...
+                    ffprobe,
+                    "-v",
+                    "error",
+                    "-select_streams",
+                    "v:0",
+                    "-show_entries",
+                    "stream=pix_fmt",
+                    "-of",
+                    "default=nokey=1:noprint_wrappers=1",
+                    absolute_input_path,
```
Comment (rsanchez87) [inline scripts/gen_jvt.py:165]: Why the last "," ?

**Review by rsanchez87 — COMMENTED** [review]
Everything looks good, just a few comments

File: scripts/gen_jvt.py (hunk 1 above)
Comment (mdimopoulos) [reply scripts/gen_jvt.py:292]: It's a matter of automatic syntax correction that black formatter does.
I will try to unify

File: scripts/gen_jvt.py (hunk 2 above)
Comment (mdimopoulos) [reply scripts/gen_jvt.py:189]: It's a matter of automatic syntax correction that black formatter does.

File: scripts/gen_jvt.py (hunk 3 above)
Comment (mdimopoulos) [reply scripts/gen_jvt.py:302]: Same

File: scripts/gen_jvt.py (hunk 4 above)
Comment (mdimopoulos) [reply scripts/gen_jvt.py:84]: Same

File: scripts/gen_jvt.py (hunk 5 above)
Comment (mdimopoulos) [reply scripts/gen_jvt.py:165]: Same

**Review by rsanchez87 — DISMISSED** [review]

**Comment by rsanchez87** [comment]
@mdimopoulos  Take care, CI is failing

**Review by rsanchez87 — APPROVED** [review]

File: test_suites/h264/JVT-AVC_v1.json [hunk 6]
```diff
@@ -1,7 +1,7 @@
 {
-    "name": "JVT-AVC_V1",
+    "name": "JVT-AVC_v1",
```
Comment (rgonzalezfluendo) [inline test_suites/h264/JVT-AVC_v1.json:2]: This change broke the BC.

Comment (rsanchez87) [reply test_suites/h264/JVT-AVC_v1.json:2]: It's true, PR to fix:

https://github.com/fluendo/fluster/pull/202

Comment (mdimopoulos) [reply test_suites/h264/JVT-AVC_v1.json:2]: Sorry, my thinking didn't go that far.
So the names of test suites should be fixed until we make a new major release?
I guess new test suites should not be added to the package without incrementing the versions, right?

Comment (rgonzalezfluendo) [reply test_suites/h264/JVT-AVC_v1.json:2]: +1
We need start doing fluster releases.

Comment (rsanchez87) [reply test_suites/h264/JVT-AVC_v1.json:2]: [https://github.com/fluendo/fluster/pull/202](https://github.com/fluendo/fluster/pull/202) merged

**Pull request merged**